### PR TITLE
Each cable coil color now has inhands

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
imageadd: Cable coils, cablecuffs, and zipties now all have properly colored inhands.
/:cl:

### CODE REVIEW

**Comment by Bawhoppen** [comment]
ok :+1:

**Comment by Incoming5643** [comment]
I'm gonna let the changelog slide due to the rare correct usage of imageadd

**Comment by PKPenguin321** [comment]
\>inb4 we need another inhands refactor due to inhand files bloat

**Comment by ChangelingRain** [comment]
We really actually do! One for weapons, one for tools, and one for the misc stuff would make it cleaner.

**Comment by PKPenguin321** [comment]
we need a dmi decompiler tool that separates every icon state into their own dmi tbh

File: code/game/objects/items/weapons/handcuffs.dm [hunk 1]
```diff
@@ -80,7 +80,6 @@
 	name = "cable restraints"
 	desc = "Looks like some cables tied together. Could be used to tie something up."
 	icon_state = "cuff_red"
-	item_state = "coil_red"
```
Comment (KorPhaeron) [inline code/game/objects/items/weapons/handcuffs.dm]: Probably be safer to keep the icon just incase someone does something dumb and spawns this down the line

**Comment by KorPhaeron** [comment]
Conflict avoiding

**Pull request merged**